HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+Normalized Difference Vegetation Index (NDVI) implementation also acts as an objective worth investigating, in no small part because it expands our list of known and utilized input metrics, enabling implementation of similar for a wider range of future projects. 
+SPEX is yet to use near-Infrared imaging, and this analysis for NDVI serves as a new but relatively simple processing exercise.
+ 
+In addition to NDVI studies of vegetation density in the VNIR band, WUAP2 aims to demonstrate CV techniques such as object detection and image segmentation of populated areas. 
+On the ground, methods of determining and mapping vegetation density from RGB visible images alone will be developed using computer vision and machine learning techniques with public NDVI data and aerial photography, and then applied to WUAP2 datasets. 
+
+\section{Benefit to SPEX}
+\label{benefittospex}
+Having a reliable, modular and iterated HAB platform is absolutely invaluable to SPEX. First and foremost, HAB is a target worth hitting in its own right. 
+The high altitude balloon system offers a unique environment to develop skills and demonstrate prowess to RIT faculty and staff, potential SPEX members, and potential sponsors.
+A HAB bearing the mark of RIT SPEX is a declaration that we can confidently launch non-trivial payloads in a system of our making and have enough faith in our secure apparatus to link ourselves so. 
+HAB demands careful planning, intelligent utilization of resources, considerable multidisciplinary engineering and long-term teamwork. 
+A successful HAB is thus linked to a success-ready SPEX, whose members are now better prepared for any task. 
+HAB also serves as a unique testing platform for projects involving different temperatures, altitudes or pressures than those experienced normally. 
+This is particularly useful for lower-altitude rocket payloads, testing them for a fraction of the cost or risk. 
+HABs designed with a payload form resembling a cubesat or combination thereof have an even wider utilization. 
+HABs are also the easiest platform for testing of longer-range communication systems.
+
+The pursuit of HAB is the pursuit of maximizing scientific output using off-the-shelf components and present limitations of SPEX's human capital. 
+Progress here develops SPEX's knowledge base for all future missions, including SPEX's understanding of what can and cannot be achieved. 
+Presently proposed operations include rigorous pre-flight testing for diagnosis and correction of software and hardware components, post-flight analysis and amelioration and inculcation of this into the design cycle. 
+Potential payloads also include ``training'' for payloads utilizing machine learning. 
+All aspects SPEX greatly benefits from having.
+
+Another aspect to SPEX is the learning in the process, with documentation specified as a deliverable. 
```

Comment:
Documenting training sets and linking them to performance is a potential avenue for exploration. This can be evaluated both to determine which training data results in the best performance and in gauging how resilient methods implemented are to a varied number of incorrect samples in the data set.